bin/p2-preparer/main.go

```diff
@@ -15,6 +17,7 @@ import (
 )
 
 func main() {
+	kingpin.Parse()
```

Comment:
do you anticipate that someone reading this code might be confused at "there are no kingpin flags in this file! what is going on! why is it being parsed?!" ? Maybe something to explain if so.